tests/Helper/LoggerBuilder.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Lib\Tests\Helper {
+
+    use DateTimeZone;
+    use Monolog\Formatter\LineFormatter;
+    use Monolog\Handler\StreamHandler;
+    use Monolog\Logger;
+
+    /**
+     * The LoggerBuilder builds default logger for the tests.
+     * @package Lib\Tests\Helper
+     */
+    class LoggerBuilder
```

Comment:
This is not a builder pattern. Either create a real builder or rename the class to a `LoggerFactory`.